pyramid/config/views.py

```diff
@@ -1028,17 +1031,15 @@ def _check_view_options(self, **kw):
             raise ConfigurationError('Unknown view options: %s' % (kw,))
 
     def _apply_view_derivers(self, info):
-        d = pyramid.config.derivations
-        # These derivations have fixed order
+        d = pyramid.viewderivers
+
+        # These derivers are not really derivers and so have fixed order
         outer_derivers = [('attr_wrapped_view', d.attr_wrapped_view),
                           ('predicated_view', d.predicated_view)]
-        inner_derivers = [('mapped_view', d.mapped_view)]
```

Comment:
This is a change I personally struggled with when I was working on this branch. I added `inner_derivers` back at one point after discussing it with you @mmerickel, but ultimately it makes sense that even this can be overridden at will.